keyboards/atset/at2/readme.md

```diff
@@ -0,0 +1,25 @@
+# ATSET AT2
+
+A special 2 key keyboard powered by QMK.
+
+* Keyboard Maintainer: [ATSET] (https://github.com/anubhavd7)
+* Hardware Supported: RP2040 (https://www.raspberrypi.com/documentation/microcontrollers/rp2040.html)
```

Comment:
```suggestion
* Keyboard Maintainer: [ATSET](https://github.com/anubhavd7)
* Hardware Supported: RP2040
```
This is not what I asked.

Reply:
THEN WHAT I HAVE TO DO SIR?

Reply:
This keyboard is under process, sir that is why i have already uploaded the file in qmk so that i dont have to wait for it to be added and i have to even add the file on via for my keyboard name. My keyboard is currently hand wired else i would have provided you the link and image of it

Reply:
I have added the image sir you can please check it

Reply:
> My keyboard is currently hand wired

Then this should have been added to the `keyboards/handwired` directory, or the PR put on hold till a non-handwired version exists.

Reply:
i have already added a non hand wired one sir u can see the image of it , already uploaded

Reply:
> My keyboard is currently hand wired else i would have provided you the link and image of it

Then why are you contradicting yourself here?

Reply:
it has been made by my team sir, i I apologize for the mistake that has been made